src/commponents/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo } from 'react';
+import cn from 'classnames';
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+interface LoadingTodo {
+  [key: number]: number;
+}
+
+interface Props {
+  filterType: Status;
+  onFiltered: (filter: Status) => void;
+  todos: Todo[];
+  loadedDeleteTodo: () => void;
+  setTodoLoading: React.Dispatch<React.SetStateAction<LoadingTodo>>;
+}
+
+export const Footer: React.FC<Props> = props => {
+  const { filterType, onFiltered, todos, loadedDeleteTodo, setTodoLoading } =
+    props;
+
+  const countTodo = useMemo(
+    () => todos.filter(todo => !todo.completed).length,
+    [todos],
+  );
+
+  const onTodoCompleted = useMemo(
```

Comment:
`on`?why? 
i see that this variable contains boolean value and doesn't change anything